vercel.json

```
@@ -375,6 +375,14 @@
     {
       "source": "/platforms/node/guides/connect/event-processors/",
       "destination": "/platforms/node/enriching-events/event-processors/"
+    },
+    {
+      "source": "/platforms/javascript/sourcemaps/uploading/hosting-publicly/",
+      "destination": "/platforms/javascript/sourcemaps/troubleshooting_js/#multiple-origins"
+    },
```

I think this is supposed to be a redirect for the multiple-origins page, yes?

```suggestion
    {
      "source": "/platforms/javascript/sourcemaps/uploading/multiple-origins/",
      "destination": "/platforms/javascript/sourcemaps/troubleshooting_js/#multiple-origins"
    },
```

---

Phew thanks for catching this!